extensions/wikia/FilePage/FilePageHooks.class.php

```diff
@@ -199,6 +199,10 @@ public static function onUndeleteComplete( Title $title ) {
 	 * @return true -- because it's hook
 	 */
 	public static function onArticleSave( WikiPage $page ) {
+		if(get_class($page) == "WikiFilePage" && $page->getFile()->media_type &&
+		   $page->getFile()->media_type == "VIDEO" ) {
```

Comment:
https://github.com/Wikia/app/blob/dee8bc271b635de7a40ac0eb7cc39af2428ffe16/includes/Defines.php#L109

Reply:
and
```php
$page->getMediaType() ==MEDIATYPE_VIDEO
```